x/stake/handler_test.go

```diff
@@ -17,24 +17,24 @@ import (
 //______________________________________________________________________
 
 func newTestMsgCreateValidator(address sdk.ValAddress, pubKey crypto.PubKey, amt int64) MsgCreateValidator {
-	return types.NewMsgCreateValidator(address, pubKey, sdk.Coin{"steak", sdk.NewInt(amt)}, Description{})
+	return types.NewMsgCreateValidator(address, pubKey, sdk.NewCoin("steak", sdk.NewInt(amt)), Description{})
 }
 
-func newTestMsgDelegate(delAddr sdk.AccAddress, valAddr sdk.ValAddress, amt int64) MsgDelegate {
+func newTestMsgDelegate(delegatorAddr sdk.AccAddress, validatorAddr sdk.ValAddress, amt int64) MsgDelegate {
 	return MsgDelegate{
-		DelegatorAddr: delAddr,
-		ValidatorAddr: valAddr,
-		Delegation:    sdk.Coin{"steak", sdk.NewInt(amt)},
+		DelegatorAddr: delegatorAddr,
+		ValidatorAddr: validatorAddr,
+		Delegation:    sdk.NewCoin("steak", sdk.NewInt(amt)),
 	}
 }
 
-func newTestMsgCreateValidatorOnBehalfOf(delAddr sdk.AccAddress, valAddr sdk.ValAddress, valPubKey crypto.PubKey, amt int64) MsgCreateValidator {
+func newTestMsgCreateValidatorOnBehalfOf(delegatorAddr sdk.AccAddress, validatorAddr sdk.ValAddress, validatorPubKey crypto.PubKey, amt int64) MsgCreateValidator {
 	return MsgCreateValidator{
 		Description:   Description{},
-		DelegatorAddr: delAddr,
-		ValidatorAddr: valAddr,
-		PubKey:        valPubKey,
-		Delegation:    sdk.Coin{"steak", sdk.NewInt(amt)},
+		DelegatorAddr: delegatorAddr,
+		ValidatorAddr: validatorAddr,
+		PubKey:        validatorPubKey,
```

Comment:
Undo all these rename changes in this test file - this is a refactor which doesn't belong in this PR